ethereum/contracts/pyth/PythState.sol

```diff
@@ -8,12 +8,22 @@ import "./PythInternalStructs.sol";
 contract PythStorage {
     struct State {
         address payable wormhole;
-        uint16 pyth2WormholeChainId;
-        bytes32 pyth2WormholeEmitter;
+        uint16 pyth2WormholeChainId; // Superseded by validDataSources/isValidDataSource
```

Comment:
To me it meant that the newer structures serve the same function and replace these two. I'm keeping them because of  memory layout requirements of the Proxy

Reply:
Just came to my mind:
can you rename it to `_depceratedPyth2WormholeChainId` or something like this? rename is ok with the proxy upgrades.

Reply:
Will do!

Reply:
thank you ser!